README.md

```diff
@@ -82,7 +84,7 @@ Other Style Guides
     console.log(foo, bar); // => 1, 9
     ```
 
-    - Symbols and BigInts cannot be faithfully polyfilled, so they should not be used when targeting browsers/environments that don’t support them natively.
+  - Symbols and BigInts cannot be faithfully polyfilled, so they should not be used when targeting browsers/environments that don’t support them natively.
```

Comment:
this change is incorrect, as it causes this note underneath 1.1 to be nested at the same level as 1.1 and 1.2. Please revert it.

Reply:
I Just Changed the Branch and just replaced <br /> with &nbsp at ; #2467

Reply:
The change should have been made here; please never ever ever open redundant PRs.